src/Persistence/Sql.php

```diff
@@ -628,8 +632,16 @@ public function action(Model $model, $type, $args = [])
                         ->addMoreInfo('action', $type);
                 }
 
-                $fx = $args[0];
-                $field = is_string($args[1]) ? $model->getField($args[1]) : $args[1];
+                [$fx, $field] = $args;
+
+                if (is_string($field)) {
+                    if ($model->hasField($field)) {
+                        $field = $model->getField($field);
+                    } else {
+                        $field = new Expression($field);
```

Comment:
real usecases? we probably need to support args as well

Reply:
There definitely are usecases for Expression.

Reply:
but it need args!

Reply:
Reverted to original functionality

Reply:
what is currently the best alternative /wo the planned optional trait?